src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java

```diff
@@ -23,6 +23,22 @@
     @Override
     public void configure() {
 
+        from("rest:POST:/channel/workflow/")
+                .id("workflow-start")
+                .log(LoggingLevel.INFO, "## Starting new workflow")
+                .process(e -> {
+
+                    JSONObject variables = new JSONObject(e.getIn().getBody(String.class));
+
+                   zeebeClient.newCreateInstanceCommand()
+                           .bpmnProcessId(e.getIn().getHeader("processId", String.class))
```

Comment:
Are you expecting the processId as a header or query parameter? Later is expected.

Reply:
query parameter, but in camel query parameter is stored in header only. I have tested it so its working with query parameter.

Reply:
Yeah. This requires to be a path param with {BPMN_PROCESS_ID}. And add the comments, rest is fine.

Reply:
done